.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -1,17 +1,42 @@
 name: Feature Request
 description: Request a feature for Dafny
-title: "[Feature request]: "
-labels: []
+title: ""
+labels: [kind: enhancement]
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for your feature request!
+        <!--
+        Markdown is supported.
+        -->
   - type: textarea
-    id: what-do-you-want
+    id: summary
     attributes:
-      label: What is the feature you would like to see in a future version of Dafny?
-      description: Describe the feature here. Markdown is supported.
-      placeholder: I would like to have X
+      label: Summary
+      description: Brief summary of what this proposal is about.
+    validations:
+      required: false
+  - type: textarea
+    id: motivation
+    attributes:
+      label: Background and Motivation
+      description: What is the problem we are solving and in what context did you encounter it?
+    validations:
+      required: true
+  - type: textarea
+    id: proposal
+    attributes:
+      label: Proposed Feature
+      description:
+        Please provide a sketch of the feature you are proposing. Be as specific as you can: the more specific the proposal, the easier the process will be. Including screenshots of some of the existing problems can also help a lot here.
```

Comment:
I'm pretty sure it's not possible add screenshots in the template version of the issue. Have you tried?

Reply:
You might want to add "if you want to add screenshot, use the free-form issue form"